ts/DataGrid/Header/TableHeader.ts

```diff
@@ -99,11 +109,11 @@ class TableHeader {
             return;
         }
 
-        // TODO: render column grouping headers
-        // this.rows[0].renderGroupedColumnHeaders();
-
-        // render basic column headers
-        this.rows[0].render();
+        for (let i = 0, iEnd = this.levels; i < iEnd; i++) {
+            const lastRow = new HeaderRow(vp, i + 1); // Avoid indexing from 0
```

Comment:
Why do we need indexes in the header rows? Won't they get mixed up with table row indexes?

Reply:
Its required to set correct level of `aria-rowindex`